doc/api/util.md

```diff
@@ -1024,6 +1024,235 @@ Otherwise, returns `false`.
 See [`assert.deepStrictEqual()`][] for more information about deep strict
 equality.
 
+## Class: `util.MIMEType`
```

Comment:
I think it should be named `MediaType`, as Media type seems to be the preferred over MIME type (https://www.iana.org/assignments/media-types/media-types.xhtml, https://en.wikipedia.org/wiki/Media_type)

Reply:
`MIMEType` is also close to `MimeType`, which is a deprecated Web API: https://developer.mozilla.org/en-US/docs/Web/API/MimeType